src/container/streamAnalyticsManager.ts

```diff
@@ -74,11 +163,32 @@ export class StreamAnalyticsManager {
                 }
             }
         } catch (error) {
-            error.message = `Cannot parse Stream Analytics publish job information: ${error.message}`;
+            error.message = `Parse Stream Analytics jobs info failed: ${error.message}`;
             throw error;
         }
     }
 
+    private async getJobSubscription(ASAInfo): Promise<AzureSubscription> {
+        for (const azureSubscription of this.azureAccount.subscriptions) {
+            if (ASAInfo.ASAJobResourceId.indexOf(azureSubscription.subscription.id) >= 0) {
+                return azureSubscription;
+            }
+        }
+
+        const ASAJobName: string = ASAInfo.ASAJobResourceId.split("/").pop();
```

Comment:
Please add this case into test plan to validate the senario

Reply:
OK, no problem